checklists/lz_checklist.en.json

```diff
@@ -1207,11 +1213,19 @@
     {
       "category": "Security",
       "subcategory": "Secrets",
-      "text": "Use a vault per application per environment per region.",
+      "text": "Don't share Key Vault instances between applications to avoid secret sharing across environments.",
       "guid": "91163418-2ba5-4275-8694-4008be7d7e48",
       "severity": "Medium",
       "link": "https://docs.microsoft.com/azure/key-vault/general/best-practices"
     },
+    {
+      "category": "Security",
+      "subcategory": "Secrets",
+      "text": "Don't use centralized instances of Key Vault for application keys or secrets.",
+      "guid": "43276d8b-dc05-45bc-b619-e8a13f988795",
+      "severity": "Medium",
+      "link": "https://docs.microsoft.com/azure/key-vault/general/best-practices"
+    },

```

Comment:
Same here, I think this was removed over in a recent PR, since it is redundant with the previous check.

Reply:
removed whole check.